tdrs-backend/tdpservice/users/api/login.py

```diff
@@ -39,13 +39,16 @@ class UnverifiedEmail(Exception):
 
     pass
 
+class ACFUserLoginDotGov(Exception):
+    """Exception for catching ACF Users using Login.gov."""
```

Comment:
@abottoms-coder what is the expected behavior when someone with a PIV/CAC uses the `Sign in with your government employee ID` (instead of u/p) option via login.gov? because I was able to successfully use the external user route.

Reply:
The way we catch information here is based on the e-mail submitted by the user -- we don't look at card vs password. We reject entirely based on e-mail since there's no metadata to describe *how* the user authenticated.